app/queries/live_edition_blocking_draft_edition.rb

```diff
@@ -12,7 +12,7 @@ def self.call(content_id, base_path, document_type)
 
       conflicts = Edition.with_document
         .where(base_path: base_path, content_store: :live)
-        .where.not("documents.content_id": content_id)
+        .where.not(documents: { content_id: content_id })
```

Comment:
This change doesn't seem to belong in this commit? At least it doesn't have anything to do with `index_with`.

Reply:
I guess you're correct, I should move that to a relevant commit.